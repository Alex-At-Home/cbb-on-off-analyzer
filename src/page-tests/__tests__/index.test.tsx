import React from 'react';
import OnOffAnalysisPage from '../../pages/index';
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

describe("OnOffAnalysisPage", () => {
  test("OnOffAnalysisPage - should create snapshot", () => {
    const wrapper = shallow(<OnOffAnalysisPage/>);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
