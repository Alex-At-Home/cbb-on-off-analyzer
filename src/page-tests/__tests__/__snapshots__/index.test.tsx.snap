// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OnOffAnalyzerPage OnOffAnalyzerPage - should create snapshot 1`] = `
<Container
  fluid={false}
>
  <ForwardRef
    noGutters={false}
  >
    <Col
      className="text-center"
      xs={12}
    >
      <h3>
        CBB On/Off Analysis Tool 
        <span
          className="badge badge-pill badge-info"
        >
          BETA!
        </span>
      </h3>
    </Col>
  </ForwardRef>
  <ForwardRef
    noGutters={false}
  >
    <HeaderBar
      common={Object {}}
      thisPage="game-"
    />
  </ForwardRef>
  <ForwardRef
    noGutters={false}
  >
    <GenericCollapsibleCard
      summary="On/Off: 2019/20  (M): on:'', auto-off, base:''"
      title="Team and Game Filter"
    >
      <GameFilter
        onChangeState={[Function]}
        onStats={[Function]}
        startingState={Object {}}
      />
    </GenericCollapsibleCard>
  </ForwardRef>
  <ForwardRef
    noGutters={false}
  >
    <GenericCollapsibleCard
      title="Team Analysis"
    >
      <TeamStatsTable
        teamStats={
          Object {
            "baseline": Object {},
            "off": Object {},
            "on": Object {},
          }
        }
      />
    </GenericCollapsibleCard>
  </ForwardRef>
  <ForwardRef
    noGutters={false}
  >
    <GenericCollapsibleCard
      title="Lineup Comparison"
    >
      <RosterCompareTable
        gameFilterParams={Object {}}
        rosterCompareStats={
          Object {
            "baseline": Object {},
            "off": Object {},
            "on": Object {},
          }
        }
      />
    </GenericCollapsibleCard>
  </ForwardRef>
  <ForwardRef
    noGutters={false}
  >
    <GenericCollapsibleCard
      title="Individual Analysis"
    >
      <RosterStatsTable
        gameFilterParams={Object {}}
        rosterStats={
          Object {
            "baseline": Object {},
            "off": Object {},
            "on": Object {},
          }
        }
        teamStats={
          Object {
            "baseline": Object {},
            "off": Object {},
            "on": Object {},
          }
        }
      />
    </GenericCollapsibleCard>
  </ForwardRef>
  <Footer
    server="server"
  />
</Container>
`;
